### ISSUE TITLE
[Feature request] Pressing Enter to paste cells stored in clipboard into spreadsheets

### ISSUE THREAD

**Issue by hfc2x** [ONLYOFFICE/DesktopEditors#1757]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

I'm trying to migrate my elderly father to ONLYOFFICE from MS Office, and he ran into a minor annoyance because this is seemingly the default behavior in MS Excel.

Whenever the user copies or cuts any range of cells in a spreadsheet, the Enter (Return) key should be able to paste the clipboard contents into the spreadsheet, instead of jumping to the next line, AKA the cell below. This is apparently expected from spreadsheet editors, which is why it's also the default behavior of LibreOffice Calc.

Pressing Enter to paste only works if the clipboard contents come from a spreadsheet, otherwise it gets ignored.

**Comment by VladimirProkofiev52** [comment]
Hello! @hfc2x 
Could you please clarify if the following workflow is what you have in mind?

- You copy data from a source that is not formatted as a table (for example, a webpage).
- Open a table in our editor.
- Select a specific cell in the table where you want the data to be inserted.
- Press the Enter key to paste the copied data directly into the selected cell.
Let us know if this is correct or if you need adjustments!

**Comment by hfc2x** [comment]
The way this normally works in MS Office and LibreOffice, is that pressing Enter on the current cell only pastes what is in the clipboard _if and only if_ the latest contents of the clipboard come from a spreadsheet document. That is, when cells are surrounded by a doted line, indicating they're currently copied or cut. This feature is commonly used to move cells within the same document.

In MS Office, when the user copies anything that does not come from a spreadsheet, pressing Enter ignores the clipboard and acts like it currently does in ONLYOFFICE.

In LibreOffice, the behavior is more complicated, sometimes using the "import" dialog when there is another type of content in the clipboard. However, in general use cases, works mostly like MS Office.

To summarize:

* If there are a range of cells in a spreadsheet either copied or cut (surrounded by dotted line), Enter should act as "paste here".
* If anything else is in the clipboard, that is not specifically from a spreadsheet (websites, tables from other documents), ignore it.
